docs/messaging/azure-service-bus-component.md

```diff
@@ -171,6 +171,8 @@ The following configurable options are exposed through the <xref:Aspire.Azure.Me
 | `ConnectionString`        | The connection string used to connect to the Service Bus namespace. |
 | `Credential`              | The credential used to authenticate to the Service Bus namespace.   |
 | `FullyQualifiedNamespace` | The fully qualified Service Bus namespace.                          |
+| `DisableHealthChecks`     | Disables health checks for the Service Bus client.                  |
```

Comment:
This isn't correct. `AzureMessagingServiceBusSettings` has:

* public string? HealthCheckQueueName
* public string? HealthCheckTopicName

You need to set one of those properties in order to enable health checks.

Reply:
To confirm, both are not required, just one or the other?

Reply:
What happens if both are provided?